### DISCUSSION TITLE
How to use the app?

### DISCUSSION THREAD

**Discussion by kerbyyy** [2bllw8/anemo#61] [Q&A]
There's no guide or anything, it just opens the app. I'm trying to look at the settings or something like that to set up the passwords and stuff like that, but as far as I can see it's only a file manager. Is there like a setup guide?

**Comment by pixel2user** [comment]
The screenshots in combination with the detailed feature list provided at https://f-droid.org/packages/exe.bbllw8.anemo/ (I installed from there) is self-explanatory to me.

But the activity exe.bbllw8.anemo.config.ConfigurationActivity isn't exposed in a obvious way for sure. I used https://f-droid.org/packages/io.github.muntashirakon.AppManager/ to launch it.

**Reply by nain-F49FF806** [reply]
@pixel2user additional tip: you can long press the app to find config activity shortcut (on most launchers).
You may be able to drag the shortcut out to your home screen too. [_aer_](https://f-droid.org/packages/alt.nainapps.aer/) has a screenshot re this.

**Reply by pixel2user** [reply]
@nain-F49FF806 Thank you very much for the suggestion! But I like the icon (left one) of my way of creating the shortcut more:
![Screenshot_20241210-083608](https://github.com/user-attachments/assets/9424f512-28c7-4633-aefd-69826df92391)
